### DISCUSSION TITLE
Anyone successfully connected to MySQL over sslmode = REQUIRED??

### DISCUSSION THREAD

**Discussion by iammiloslukic** [leafsphp/leaf#175] [Q&A]
Hi guys, 

I'm trying to connect to DB on Digital Ocean with sslmode = REQUIRED but I'm failing.

Can anyone share how it should be done?

Thanks

**Comment by mychidarko** [comment]
Hi @iammiloslukic can you share how you're trying to connect to the database?

**Reply by iammiloslukic** [reply]
> Hi @iammiloslukic can you share how you're trying to connect to the database?

The issue was that the port number is not a default one so even though you set everything right in env, port number will be 3306 so I fixed it by changing the core basically.

Anyhow, it's fixed now, I was just testing the performance of Digital Ocean against the Contabo and they were head to head basically.
